### PR TITLE
Ensure all runs have the same order so that the results are deterministic

### PR DESCRIPTION
**Pull request by ashahba**
Ensure all runs have the same order so that the results are deterministic

## Reviewer Checklist

 - [ ] Do you understand it?
 - [ ] Is it correct?
 - [ ] Is it safe?
 - [ ] Is it legally compliant?
 - [ ] Is it robust?
 - [ ] Is it simple?
 - [ ] Is it tested?
 - [ ] Is it documented?
 - [ ] Is the style consistent?

See [the reviewer guide](docs/reviews.md) for more detail on each check.

### CODE REVIEW

File: tests/e2e/test_deploy_flow.py [hunk 1]
```diff
@@ -55,9 +55,9 @@ def teardown(self):
     # TODO: find a way to test `tensorboard-bm`.
     # `tensorboard-bm` template requires the user domain to be configured
     # as a custom parameter https://github.com/IntelAI/mlt/issues/371
-    @pytest.mark.parametrize('template', filter(lambda x: os.path.isdir(
+    @pytest.mark.parametrize('template', sorted(filter(lambda x: os.path.isdir(
```
Comment (s1113950) [inline tests/e2e/test_deploy_flow.py]: This won't work with threaded tests. We determined it wasn't threading that was the problem I thought? Or are we disabling threading for now? If we're disabling them then can you add that to this PR?

Comment (ashahba) [reply tests/e2e/test_deploy_flow.py]: Yes, I think we are disabling them for now. Aren't we?

Comment (s1113950) [reply tests/e2e/test_deploy_flow.py]: @dmsuehir I've been doing other stuff and haven't fully been keeping up with the debugging on the current mlt issues; for sure threading is causing issues?

Comment (dmsuehir) [reply tests/e2e/test_deploy_flow.py]: No, I don't think we know for sure that threading is causing issues.  That horovod test was hanging in Ebi's PR and it was not threaded.

Comment (s1113950) [reply tests/e2e/test_deploy_flow.py]: @ashahba then we should close this PR, or you can keep it for now and disable threading here just as a debugging tool.

**Pull request closed without merging**